bitrisescript/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
```

Comment:
I realize this isn't unique to bitrisescript, but what's the purpose of having a docker-compose file?

Reply:
Heitor asked the same thing and.. I don't know! I'm going to add it for now because all the other scripts have it, but yeah.. maybe we can remove them all in a follow-up PR.